edit.js

```diff
@@ -1,44 +1,125 @@
+let company = document.getElementById("company");
+let startDate = document.getElementById("start-date");
+let endDate = document.getElementById("end-date");
+let description = document.getElementById("description");
+
+let isEdit = false;
+
 function showExperienceForm() {
   document.getElementById("experience-form").style.display = "block";
 }
 document.getElementById("experience-form").addEventListener("submit", function (event) {
+  if (!company.value || !startDate.value || !endDate.value || !description.value) {
```

Comment:
use validations here, for user better understanding

Reply:
update done